README.md

```diff
@@ -42,6 +42,7 @@ Turns any MySQL, PostgreSQL, SQL Server, SQLite & MariaDB into a smart spreadshe
 [<img height="38" src="https://user-images.githubusercontent.com/61551451/135263770-38e3e79d-11d4-472e-ac27-ae0f17cf65c4.png">](markdown/readme/languages/japanese.md)
 [<img height="38" src="https://user-images.githubusercontent.com/61551451/135263822-28fce9de-915a-44dc-962d-7a61d340e91d.png">](markdown/readme/languages/korean.md)
 [<img height="38" src="https://user-images.githubusercontent.com/61551451/135263888-151d4ad1-7084-4943-97c9-56f28cd40b80.png">](markdown/readme/languages/russian.md)
+[Українська](markdown/readme/languages/ukrainian.md)
```

Comment:
please follow the same pattern with other langs

Reply:
I think it's made by our team member. Right now you may remove L45 as it's included in `markdown/readme/languages/README.md`. After that, it's good to merge.

Reply:
Updated.